CarePartner/Views/FolloweeListView.swift

```diff
@@ -184,7 +185,11 @@ struct FolloweeListView: View {
         if manager.followees.isEmpty {
             LinearGradient(gradient: Gradient(colors: [Color("accent-background"), Color("accent-background").opacity(0.2)]), startPoint: .top, endPoint: .bottom)
         } else {
-            Color.white
+            if colorScheme == .light {
+                Color.white
+            } else {
+                Color.black
+            }
```

Comment:
why cannot `.primary` be used here?

Reply:
@nhamming `.primary` would give me the opposite color here. We want white in light mode and black in dark mode. `.primary` is black in light mode and white in light mode.

Reply:
@nhamming I made a custom color set for this which makes things a lil cleaner.